jwst/datamodels/container.py

```diff
@@ -26,7 +26,7 @@
 logger.addHandler(logging.NullHandler())
 
 
-class ModelContainer(JwstDataModel, Sequence):
+class ModelContainer(JwstDataModel, Sequence, ModelList):
```

Comment:
Ahh, here is where `Sequence` is still present. As per offline discussion, since `Sequence` was just a marker, I think it should be removed from here.